### ISSUE TITLE
TypeError: croner_1.Cron is not a constructor

### ISSUE THREAD

**Issue by tukusejssirs** [Hexagon/croner#7]
I tried run the following code in TypeScript (specifically in a Nest.js service):

```ts
import {Cron} from 'croner'

const job: Cron = new Cron('* * * * * *', () => {
	console.log('This will run every second.')
})
```

But it fails with `TypeError: croner_1.Cron is not a constructor`.

If I remove `new` keyword, it fails with `TypeError: croner_1.Cron is not a function` (obviously).

I have no idea why this fails.

---

MWE example Nest.js service:

```ts
import {Injectable, Logger} from '@nestjs/common'
import {Cron} from 'croner'

@Injectable()
export class CronService {
	private readonly logger = new Logger(CronService.name)
	private jobs: Cron[] = []

	createJob(time: string) {
		// time = '*/1 * * * * *'
		const job: Cron = new Cron(time, () => {
			this.logger.debug('This will run every second.')
		})

		this.jobs.push(job)
		return job
	}
}
```

---

In CommonJS, it works as expected.

---

This might not be an issue with `croner`, but with Nest.js and how it does in the background (see [SO answer](https://stackoverflow.com/questions/57007200/unit-testing-nestjs-mongoose-services/58555364#58555364)).

---

**jmcdo29** at Nest.js Discord Channel found out that the issue is in `croner`’s typing, citing:

> Looks like it's bad typings on `croner`'s part. I had to do
>
> ```ts
> import * as Cron from 'croner';
> (Cron as any)('* * * * * *', () => console.log('run every second'));
> ```
>
> To get it to work

**Comment by Hexagon** [comment]
Not familiar with Nest.js, but i will have a look at it when i fint the time to do it.

A quick guess:

Normally (if following package.json), /src/croner.js or /dist/croner.min.mjs should be used when using import. Those are pure "modules".

Could it be that Nest.js is trying to use the commonjs version (/dist/croner.cjs or /dist/croner.min.js) and convert it to an es module on it's own?

**Comment by Hexagon** [comment]
Turns out that typescript uses the commonjs version of this module internally, and the commonjs code only has a single export (default). That's why only ```import Cron from 'croner';``` works.

The default typings indicated that both named and default export were available, that has been fixed too.

Also clarified TypeScript usage in README.me

**Comment by Hexagon** [comment]
Fixed in 4.0.75

**Comment by Hexagon** [comment]
Fixed in 4.0.76

**Comment by tukusejssirs** [comment]
Sorry, @Hexagon, for being late. Yes, now it works without issue. Thanks! :pray:
